tracker/tests/setup_single_node.sh

```diff
@@ -0,0 +1,24 @@
+PORT=$1
```

Comment:
Could you please add an example in README on how to run this script?

Reply:
I have added description and examples in 94c56bb59e24c4cd880602d7581c29d4a8b9eaf8. Now is it clear now how to use the script?